org.eclipse.lsp4e/src/org/eclipse/lsp4e/LSPEclipseUtils.java

```diff
@@ -333,7 +334,12 @@ public static CallHierarchyPrepareParams toCallHierarchyPrepareParams(int offset
 		Position position =  LSPEclipseUtils.toPosition(offset, document);
 		TextDocumentIdentifier documentIdentifier = toTextDocumentIdentifier(document);
 		return new CallHierarchyPrepareParams(documentIdentifier, position);
+	}
 
+	public static TypeHierarchyPrepareParams toTypeHierarchyPrepareParams(int offset, final @NonNull IDocument document) throws BadLocationException {
```

Comment:
If this is only used in TypeHierarchyContentProvider, let's just move it there. LSPEclipseUtils is more meant for reusable bits that several operations require.